evaluation.py

```diff
@@ -29,19 +29,26 @@ def write_predictions(predictions, checkpoint_path):
         for sentence in predictions:
             out_file.write(sentence + '\n')
 
-def idx_to_TOKENs(predictions, trg_vocab):
+def idx_to_TOKENs(predictions, trg_vocab, src_sents = None, copy_lut = None, attn = None):
```

Comment:
What's the difference between _idxs_to_sentences and idx_to_TOKENs? If  they provide a similar functionality, we should just keep one.

Reply:
I just mean if there was something subtly different between the two. You can just offload this to me if you want since I accidentally wrote the duplicate.

Reply:
Is it a member function of the model? That might be interesting place to convert idxs to sentences, rather than outside the model. Only downside I see is duplicate code for each model (unless we use inheritance).